client/templates/admin/postEditor/postEditor.html

```diff
@@ -1,14 +1,12 @@
 <template name="postEditor">
-  <div class="panel panel-default">
+
     {{ #if isReady }}
 
       <div class="panel-heading">
         {{ #if isEditing }}
-          <h2 class="pull-left">Edit post</h2>
-          <button class="btn btn-primary pull-right savePost">Save post</button>
+          <button class="update-post-button savePost">Oppdater</button>
         {{ else }}
-          <h2 class="pull-left">New post</h2>
-          <button class="btn btn-primary pull-right savePost">Save new post</button>
+          <button class="add-btn savePost">Save new post</button>
```

Comment:
Good point! Will do that. 

> 16. mai 2016 kl. 16.04 skrev Sebastian Sastre <notifications@github.com>:
> 
> In client/templates/admin/postEditor/postEditor.html <https://github.com/dagvonkr/hof3/pull/10#discussion_r63357543>:
> 
> >          {{ else }}
> > -          <h2 class="pull-left">New post</h2>
> > -          <button class="btn btn-primary pull-right savePost">Save new post</button>
> > +          <button class="add-btn savePost">Save new post</button>
> @dagvonkr <https://github.com/dagvonkr> you translated the save existing post, don't you want to translate also the save new post?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/dagvonkr/hof3/pull/10/files/98f9d483bafb635547f55853ca92fc8d1d9eb88c#r63357543>